free-programming-books-ru.md

```diff
@@ -203,12 +205,15 @@
 * [Руководство по Java Core](http://proselyte.net/tutorials/java-core) - Евгений Сулейманов
 * [Руководство по Servlets](http://proselyte.net/tutorials/servlets) - Евгений Сулейманов
 * [Руководство по языку программирования Java](https://metanit.com/java/tutorial) - Евгений Попов
+* [Собеседование по Java Core](http://javastudy.ru/interview/list-of-question-java-interview)
```

Comment:
I guess it's sort of part textbook, part course, part study material?

Reply:
Good quality tutorials